conjurapi/resource.go

```diff
@@ -83,3 +83,19 @@ func (c *Client) Resources(filter *ResourceFilter) (resources []map[string]inter
 
 	return
 }
+
+func (c *Client) ResourceIDs(filter *ResourceFilter) ([]string, error) {
+	resources, err := c.Resources(filter)
```

Comment:
Curious if selecting just the `id` field is something that can be done at the API level...

Reply:
What do you mean? This is the API. :)